configuration.go

```diff
@@ -52,6 +55,10 @@ func NewConfiguration() (config *Configuration, err error) {
 		return nil, err
 	}
 
+	if config.PrecedingUpdateRequireInitials, err = strconv.ParseBool(getenvDefault("GIT_DUET_PRECEDING_UPDATE_REQUIRE_INITIALS", "0")); err != nil {
```

Comment:
I'm wondering if we could just call the environment variable `GIT_DUET_DEFAULT_UPDATE`. I think that is a little more clear.

Reply:
Sounds better. I'll find my time to update the PR later.

Reply:
Changed the environment variable name :+1: